frontend/src/store/wallet.ts

```diff
@@ -477,21 +477,20 @@ export default function useWalletStore() {
   });
 
   const tokens = computed((): TokenInfoExtended[] => {
-    let tokensArray: TokenInfoExtended[] = [];
+    // Sort alphabetically.
+    const initialTokens = relayer.value?.tokens || [];
+    let tokensArray = initialTokens.sort((a, b) => a.symbol.localeCompare(b.symbol));
```

Comment:
RR: If I'm reading this PR right, we're sorting alphabetically, then splicing the native token out, then putting the native token back in at the front of the array. Could we simplify this by customizing the token sorting function here to return the Native Token ahead of other tokens?

Reply:
> If I'm reading this PR right, we're sorting alphabetically, then splicing the native token out, then putting the native token back in at the front of the array.

Exactly

> Could we simplify this by customizing the token sorting function here to return the Native Token ahead of other tokens?

You definitely can do that, I'm not sure if it's actually simpler though, since then the flow is: check for native token, add ours if it's missing, then sort. So it's still 3 similar steps just in a different order

Reply:
Clearer code introduced in 16927f5

Reply:
Much cleaner! Thank you for humoring my suggestion :sweat_smile: